### PR TITLE
Fixed initial sorting of datagrid

### PR DESCRIPTION
**Pull request by michaelzangerle**
__Tasks:__
- [x] done
- [ ] got feedback

__Informations:__

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Tests pass?   | none
| Fixed tickets | none
| Doc           | none

### CODE REVIEW

File: dist/husky.js [hunk 1]
```diff
@@ -51124,6 +51152,19 @@ define('husky_extensions/util',[],function() {
                 return $.extend(true, parent, object);
             };
 
+            /**
+             * Returns a parameter value from a given url
```
Comment (wachterjohannes) [inline dist/husky.js]: can you give me the link where this comes from?

File: dist/husky.js [hunk 2]
```diff
@@ -51124,6 +51152,19 @@ define('husky_extensions/util',[],function() {
...
+             * @param name {string} name of the parameter to search for
+             * @param url {string}
+             * @returns {string}
+             */
+            app.core.util.getParameterByName =  function(name, url) {
+                name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+                var regex = new RegExp("[\\?&]" + name + "=([^&#]*)"),
+                    results = regex.exec(url);
+                return results === null ? "" : decodeURIComponent(results[1].replace(/\+/g, " "));
```
Comment (wachterjohannes) [inline dist/husky.js]: empty line before return

File: dist/husky.js (hunk 1 above)
Comment (michaelzangerle) [inline dist/husky.js]: http://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript

File: dist/husky.js (hunk 2 above)
Comment (michaelzangerle) [inline dist/husky.js]: added it in the correct file :)

**Pull request merged**